### PR TITLE
Feature/cc123 My Account Page

### PR DESCRIPTION
**Pull request by wwwseven**
- ### Create my account page

- ### Add test for my account page

account

### CODE REVIEW

File: src/components/MyAccount/MyAccountContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+              background="#F3F2F7"
+              border="1px solid #344C5C"
+            >
+              Change Password
+            </Button>
+          }
+        />
+      </Box>
+      <Button variant="shareBtn" padding="10px 24px" onClick={handleReturnToDesign}>
+        Return To Design
```
Comment (zhuocun) [inline src/components/MyAccount/MyAccountContainer.tsx:90]: A minor one, may be "To" could be "to"

**Review by zhuocun — APPROVED** [review]
LGTM

**Review by GLC-coder — APPROVED** [review]
Good job!

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**